release-notes.md

```diff
@@ -8,6 +8,43 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 Please take a moment to fill out a [short-survey](https://www.surveymonkey.com/r/AEPDocs) on how we can better assist you with enabling Adobe Experience Cloud solutions and services on your mobile apps.
 
+## January 13, 2020
+
+The following updates were made in this release:
+
+**iOS Griffon 1.0.3**
+
+* Griffon bridge and Griffon SDK unified. Setup steps are slightly different now. See [here](https://aep-sdks.gitbook.io/docs/beta/project-griffon/set-up-project-griffon) for updated documentation.
+* Adobe Analytics debug flag now enabled on Griffon session start and disabled on session stop.
+* Client-side UI now logs location service events (entry/exit) events.
+* Updated for iOS 13 compatibility.
+* Fixed a crash of LIBDISPATCH.
+
+**Android Griffon 1.1.0**
+* Griffon bridge and Griffon SDK unified. Setup steps are slightly different now. See [here](https://aep-sdks.gitbook.io/docs/beta/project-griffon/set-up-project-griffon) for updated documentation.
+* Adobe Analytics debug flag now enabled on Griffon session start and disabled on session stop.
+* Client-side UI now logs location service events (entry/exit) events
+* Modified client-ui to include client-side logging capabilities.
+* Fixed an issue for Android API versions 28 or later where multiple WebViews cannot share the same data directory.
+* Added generic exception handling for exceptions that can occur when WebViews are accessed while OS is updating Chrome.
+
+The Project Griffon web UI now has new views specifically for users trying to inspect and improve Adobe Analytics and Location Service (Places) implementations.
+
+**Adobe Analytics View**
+The new Adobe Analytics view shows you events only related to your Adobe Analytics implementation. The list view now shows the action/state name and event “status” along with a newly formatted detail view. Status tells you when an SDK event is generated (processed), if the SDK has made a network request with Adobe Analytics (queued), and if we’ve returned post-processing information about that event (validated). This information should give you more confidence to determine if your context data is being mapped appropriately in Adobe Analytics.
```

Comment:
- Please add a line space between the heading and the text.

- Please reword the section as follows:
"The new Adobe Analytics view shows you events that are only related to your Adobe Analytics implementation. The list view now displays the action/state name and event, `status`, with a newly formatted detail view. `Status` tells you when an SDK event is generated (processed), whether the SDK has made a network request with Adobe Analytics (queued), and whether post-processing information about the event (validated) is returned. This information helps you determine whether your context data is being appropriately mapped in Adobe Analytics.